content/kubermatic/master/guides/telemetry/_index.en.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Telemetry"
+date = 2021-09-10T08:07:15+02:00
+weight = 20
+
++++
+
+Telemetry is an observability tool that can be used to track Kubermatic Kubernetes Platform and Kubernetes cluster usage. It collects anonymous data and helps us to improve KKP performance for large and scalable setups. The following guide explains how to enable and disable the Telemetry tool in your KKP installation, and what kind of data it collects.
+
+Telemetry helm chart can be found in the [Kubermatic repository](https://github.com/kubermatic/kubermatic/tree/master/charts/telemetry), and since Telemetry is an open source tool, the code can be found in [Telemetry-Client repository](https://github.com/kubermatic/telemetry-client).
+
+## Installation
+### Kubermatic installer
+Telemetry will be enabled by default if you use the Kubermatic installer to deploy KKP. For more information about how to use the Kubermatic installer to deploy KKP, please refer to the [installation guide]({{< relref "../installation/" >}}).  
+Kubermatic installer will use a `values.yaml` file to configure all Helm charts, including Telemetry. The following is an example of configuration of Telemetry tool:
```

Comment:
NIT: Replace `The following is an example of configuration of Telemetry tool` to
`The following is an example of the configuration of the Telemetry tool`